README.md

```diff
@@ -11,7 +11,7 @@ Bindings / wrapper API for libclang in the [D programming language](https://dlan
 Build Instructions
 ------------------
 
-To reproduce the environment of dpp's CI, you can run [setup-cpp](https://github.com/aminya/setup-cpp) with `--llvm=11.0.0` and `--vcvarsall=true`. This will set up LLVM 11.0.0 and the proper environment variables.
+Note: for a reproducible and cross-platform build environment, you can run [setup-cpp](https://github.com/aminya/setup-cpp) with `--llvm=11.0.0` and `--vcvarsall=true`. This will set up LLVM 11.0.0 and the proper environment variables.
```

Comment:
Why is `--vcvarsall=true` necessary? It's especially confusing if running this on Linux.

Reply:
Because although a single set of flags are used, setup-cpp is smart enough to skip windows specific operations on other operating systems.

Reply:
That's still confusing for the user - why is the flag not defaulted to true, then? Speaking of which, why doesn't the LLVM version default to 11?

Reply:
Because the documentation of setup-cpp explains things thoroughly, I have not seen anyone getting confused. Here we lack context, and so that might be the reason. I have removed vcvarsall. 

> why doesn't the LLVM version default to 11?

setup-cpp uses the latest version, which is LLVM 13. Because you are testing with LLVM 11, I did not want to update the version.